spec/features/admin/admin_can_create_tutorial_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe 'as an admin' do
+  it 'can view new tutorial form' do
+    admin = create(:user, role: 1)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+    visit '/admin/tutorials/new'
+
+    expect(page).to have_field('Title')
```

Comment:
This test isn't actually doing anything... might have been added into a commit by mistake?

Reply:
It certainly was, this test shouldn't be here. I deleted the file on my local but must have committed it sometime before I did that.